### PR TITLE
go: Add simple server/client using raw for benchmarking

### PR DESCRIPTION
**Pull request by prashantv**
r @mmihic

### CODE REVIEW

File: golang/examples/bench/server/server.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+	ch, err := tchannel.NewChannel("bench-server", &tchannel.ChannelOptions{
+		ProcessName: "bench-server",
+	})
+	if err != nil {
+		log.Fatalf("NewChannel failed: %v", err)
+	}
+
+	handler := raw.Wrap(&kvHandler{vals: make(map[string]string)})
+	ch.Register(handler, "get")
```
Comment (Raynos) [inline golang/examples/bench/server/server.go]: can you implement ping as well to have compat with the node bench server ?

Comment (prashantv) [inline golang/examples/bench/server/server.go]: Will do in a separate PR.
